src/assets/constitutions/X7.json

```
@@ -0,0 +1,774 @@
+{
+  "cstName": "Septième Xénocratie",
```

```suggestion
  "constitution": {
     "name": "Septième Xénocratie"
     "date": "2022-05-07T16:40:50.971Z"
     "type": "Xénocratie"
     "series": 7
  }
```
on évite les variables préfixées par un sous type.

---

À noter pour plus tard je pense. Pour le moment, on est pas intéressé par les champs `type` et `series`. Septième Xénocratie est ici seulement le nom, on ne veut pas faire d'opération spéciale dessus.